### PR TITLE
On demand plus stream demo

### PR DESCRIPTION
**Pull request by franciscojavierarceo**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

### CODE REVIEW

**Comment by franciscojavierarceo** [comment]
FYI @Rostifar @hao-affirm this is just a simple demo and it looks like i found weird behavior in dates (though I guess some is kind of expected) and a bug in Feast's handling of requesting the `created` timestamp.

File: examples/quickstart/feature_repo/example.py [hunk 1]
```diff
@@ -40,38 +48,99 @@
...
+        Field(name="days_since_last_created_date", dtype=Int64),
     ],
 )
 def transformed_conv_rate(inputs: pd.DataFrame) -> pd.DataFrame:
     df = pd.DataFrame()
-    df['output'] = inputs['conv_rate'] + inputs['int_val']
-    df['time_since_last_date'] = (datetime.now() - pd.to_datetime(inputs['created'])).dt.seconds
+    df["output"] = inputs["conv_rate"] + inputs["int_val"]
+    datedelta = pd.to_datetime(datetime.utcnow(), utc=True) - pd.to_datetime(
+        inputs["created"], utc=True
```
Comment (hao-affirm) [inline examples/quickstart/feature_repo/example.py:100]: why do you use `created` timestamp, shouldn't it be the `event_timestamp`?

**Comment by franciscojavierarceo** [comment]
i can change it to event stamp, i just used created as an example

File: examples/quickstart/feature_repo/example.py (hunk 1 above)
Comment (franciscojavierarceo) [reply examples/quickstart/feature_repo/example.py:100]: updated to do both in https://github.com/Affirm/feast/pull/13

**Pull request merged**